### DISCUSSION TITLE
Russian Translation for the game

### DISCUSSION THREAD

**Discussion by ZAZiOs** [chrislo27/PolyrhythmMania#50] [Ideas]
I want to help with translating the game to Russian Language (For now im doing RHFever full translation, and its already 45% complete)

![image](https://user-images.githubusercontent.com/85897688/193037822-755e8982-1cd3-4122-bb76-4a71c73b09c6.png)

**Comment by chrislo27** [comment]
Thanks for your interest. I would like to hold off on localization until the [Version 2.0.0 major update](https://github.com/chrislo27/PolyrhythmMania/discussions/49) is released due to the large number of strings added and changed. There will be more localization utilities in that update as well.
